scripts/simulator_base.js

```diff
@@ -1117,6 +1117,24 @@ var SIMULATOR = {};
 					drawField(field, null, null, turn, sourceCard);
 				}
 			}
+		},
+
+		// Rally
+		// - Targets self
+		// - only applies to first attack
+		bash: function(src_card, activations) {
+			var rally_amt = src_card.bash;
+			if (activations === 2 || (!src_card.dualstrike_triggered)) {
```

Comment:
This will cause Frogs to have extra attack each turn, but my understanding was that it should only happen for their first attack for the entire match.

Reply:
I re-read the BGE announcement and I stand corrected - it does trigger each turn.  I updated my implementation to reflect that.